components/kyma-environment-broker/cmd/broker/broker_suite_test.go

```diff
@@ -367,7 +374,7 @@ func (s *BrokerSuiteTest) FinishProvisioningOperationByReconciler(operationID st
 func (s *BrokerSuiteTest) AssertProvisionerStartedProvisioning(operationID string) {
 	// wait until ProvisioningOperation reaches CreateRuntime step
 	var provisioningOp *internal.ProvisioningOperation
-	err := wait.Poll(pollingInterval, 2*time.Second, func() (bool, error) {
+	err := wait.Poll(pollingInterval, 2000*time.Second, func() (bool, error) {
```

Comment:
nit: could be more readable as `33 * time.Minute`

Reply:
fixed